config/routes.rb

```diff
@@ -40,6 +40,7 @@
   resource :pricing, only: [:show]
   resource :search, only: [:show], controller: :search
   resources :websites, only: [:index]
+  resources :administrator_invitations, only: [:update], path: "/administrator/invitations"
   resource :administrator_dashboards, only: [:show], path: "/dashboards/administrator"
```

Comment:
I am just going with the flow here, but I was kinda curious, why not namespace these controllers, I mean something like:-

```
namespace : administrator do
end